student.rb

```diff
@@ -1,8 +1,10 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
-    super(age, parent_permission, name)
+    super(age, name, parent_permission)
     @classroom = classroom
   end
 
```

Comment:
- [ ]  Hi, the requirement asks to **Create the belongs-to side (a student belongs to a classroom).**
- I would suggest you check, and use this[ method that serves as the 'belongs-to' side](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) (a student belongs to a classroom)

Example:
```ruby
 def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Reply:
>  - [ ] As requested by the previous reviewer, please incorporate the method representing the 'belongs-to' relationship, illustrating that a student is associated with a classroom. Refer to this [Link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for more information.👍
>  
> You may use the below example from the previous reviewer as a reference:
>  
> ```
> def classroom=(classroom)
>       @classroom = classroom
>       classroom.students.push(self) unless classroom.students.include?(self)
> end
> ```